distribution/solr-distro/src/main/resources/server/solr/security.json

```diff
@@ -0,0 +1,24 @@
+{
+  "authentication": {
+    "blockUnknown": true,
+    "class": "solr.BasicAuthPlugin",
+    "credentials": {
+      "ddf": "aMp++h2DCQ8EbF+g1Rh6pRSRrDsU+WcUZk+TNHEJ4FA= 4PZmryDswuG+Vaau+sEYIkuqY+QpCINK7QxxOWy+yAk="
+    },
+    "": {
+      "v": 0
+    }
+  },
+  "authorization": {
+    "class": "solr.RuleBasedAuthorizationPlugin",
+    "permissions": [
+      {
+        "name": "security-edit",
+        "role": "admin"
+      }
+    ],
+    "user-role": {
+      "ddf": "admin"
+    }
+  }
+}
```

Comment:
This file tells Solr to use basic auth and creates a single user named "ddf" with password "changeit"